### ISSUE TITLE
Security researcher

### ISSUE THREAD

**Issue by ruvlesh** [usebruno/bruno#1954]
Import collection failed message is displayed.
No further details provided

**Comment by samc-vaarst** [comment]
Same issue!

**Comment by philippgille** [comment]
Same here, issues with 1.13.0 and 1.13.1. Haven't tried downgrading.

**Comment by philippgille** [comment]
Probably duplicate of https://github.com/usebruno/bruno/issues/1868

**Comment by armdab** [comment]
Hello
I have same error after updating Bruno on 1.12.3 Mac x64 version.
To replicate this issue you need to export a Bruno collection and Import same Bruno collection.
<img width="956" alt="Capture d’écran 2024-04-10 à 18 11 55" src="https://github.com/usebruno/bruno/assets/45043209/869f6521-28f2-4d80-ac09-3466ef7682c3">

**Comment by armdab** [comment]
This is very critical, cause no turnover(or tell us how to by pass)... I'm blocked on my usage of BRUNO :(

**Comment by AnthonySou** [comment]
@armdab As a workaround, I simply remove the first item (the one with type=json) in the exported json file.

**Comment by ruvlesh** [comment]
@AnthonySou , can you pleas be more specific?

**Comment by AnthonySou** [comment]
@ruvlesh Steps :
- Export bruno collection
- Edit the exported json file, find the first "items" array and remove the first json object of this array (this object have a property "type"="json")
- Save the file
- Import the bruno collection and that was ok for me
Hope this will be helpful.

**Comment by anusree-bruno** [comment]
Can you try exporting from the latest version of bruno and re importing and see if the issue still persists?
